contracts/voting-verifier/src/events.rs

```diff
@@ -148,7 +147,7 @@ impl FromStr for MessageByTxEvent {
     }
 }
 
-fn parse_message_id(message_id: &MessageID) -> Result<(String, u64), ContractError> {
+fn parse_message_id(message_id: &MessageId) -> Result<(String, u64), ContractError> {
     // expected format: <source_chain>:<tx_id>:<index>
     let components = message_id.as_str().split(ID_SEPARATOR).collect::<Vec<_>>();
```

Comment:
not sure if it belongs to the scope of this pr. the function will break as message id does not contain <source_chain> anymore

Reply:
nvm, I found you updated in the following up pr